pkg/storaged/stratis/pool.jsx

```diff
@@ -109,40 +110,8 @@ function create_fs(pool) {
                                                           variant == "nomount");
                           }
                       }),
-            CheckBoxes("mount_options", _("Mount options"),
-                       {
-                           value: {
-                               ro: false,
-                               extra: false
-                           },
-                           fields: [
-                               { title: _("Mount read only"), tag: "ro" },
-                               { title: _("Custom mount options"), tag: "extra", type: "checkboxWithInput" },
-                           ]
-                       }),
-            SelectOne("at_boot", _("At boot"),
-                      {
-                          value: client.in_anaconda_mode() ? "local" : "nofail",
-                          explanation: mount_explanation.nofail,
-                          choices: [
-                              {
-                                  value: "local",
-                                  title: _("Mount before services start"),
-                              },
-                              {
-                                  value: "nofail",
-                                  title: _("Mount without waiting, ignore failure"),
-                              },
-                              {
-                                  value: "netdev",
-                                  title: _("Mount after network becomes available, ignore failure"),
-                              },
-                              {
-                                  value: "never",
-                                  title: _("Do not mount"),
-                              },
-                          ]
-                      }),
+            mount_options(false, false),
+            at_boot_input(client.in_anaconda_mode() ? "local" : "nofail"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19849-20240118-154511-b2bc543c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)